service/history/queueProcessor.go

```diff
@@ -155,8 +162,6 @@ func (p *queueProcessorBase) notifyNewTask() {
 }
 
 func (p *queueProcessorBase) processorPump() {
-	<-time.NewTimer(backoff.NewJitter().JitDuration(p.options.StartDelay(), 0.99)).C
```

Comment:
Why we don't need this jitter

Reply:
this randomized delay was added as part of 2DC, from what i have seen in the past year, this does not really do anything